articles/step-by-step-on-how-to-user-nodejs-repl-tutorial/index.md

```diff
@@ -0,0 +1,203 @@
+### Introduction
+Node.js `Read -Evaluate-Print-Loop` is an easy-to-use command-line tool for processing Node.js expressions.
+This shell captures the user's JavaScript code inputs, interprets and evaluates the result of this code, displays the result to the screen, and repeats the process till the user quits the shell.  
+
+It's therefore, important to note that this tool does not require file creation to write codes. It comes with ready to use Node.js development environment.
```

Comment:
It would be great if you could give a more in detailed introduction about REPL, why one must choose REPL, what other alternatives do they have, how does it work.

Also, give an overview of what the reader is going to learn in this article.

Refer other section articles to get a better understanding on how to write `Introduction`

Reply:
Add any link for downloading this command line tool.